source/_integrations/zha.markdown

```diff
@@ -84,6 +84,8 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
 
 ## Configuration - GUI
 
+Connect your radio module and restart Home Assistant.
```

Comment:
That should not be needed?

Reply:
Restart should not be needed with Home Assistant OS, but if not using Home Assistant OS as the installation method...

Reply:
For whatever reason, after plugging a new USB stick into Running Home Assistant OS installation for the very 1st time, it does show as a serial port in the Supervisor Hardware tab, however accessing the serial port from the HA Core instance fails. Exactly same steps work after the host reboot.